### ISSUE TITLE
Document filter textbox (SQL API) is white-on-white in dark theme

### ISSUE THREAD

**Issue by LarsKemmann** [microsoft/AzureStorageExplorer#649]
**Storage Explorer Version**: 1.4.1
**Platform/OS Version**: Windows
**Architecture**: ia32
**Regression From**: N/A

#### Steps to Reproduce: ####
1. Use dark theme
2. Try to edit a filter expression on a SQL API document collection, with the cursor hovering over the filter textbox

#### Expected Experience: ####
Black text on white, or similar

#### Actual Experience: ####
White text on white, making it impossible to read

**Comment by gruckion** [comment]
I was going to submit this issue myself. Here's an example

![KQiFuNmYgs](https://user-images.githubusercontent.com/4559650/55732858-9bb9c680-5a14-11e9-9416-8d4d0cf01264.gif)

If the text is selected it's fine, but when you hover over the text needs to turn black.

**Comment by ziyel** [comment]
Duplicate with https://github.com/microsoft/AzureStorageExplorer/issues/626.
